src/services/book.service.ts

```diff
@@ -21,9 +22,13 @@ const paramsToUrl = (booksRequest: BooksRequest) => {
 }
 
 const create = (formData: FormData)  => baseAxios.post('/Books', formData)
+const update = (formData: FormData)  => baseAxios.put('/Books', formData)
```

Comment:
i know this is obvious that you are updating the book, since this function is in book.service, but you can make it a bit more descriptive, same applies to the rest of functions in service.